machineconfig/master/spec/config/storage/files/check_ocp_api.sh L2V0Yy9rZWVwYWxpdmVkL2NoZWNrX29jcF9hcGkuc2gK

```diff
@@ -0,0 +1 @@
+exec curl -o /dev/null -sIf --cacert /etc/kubernetes/ca.crt "https://0:6443"
```

Comment:
Sorry, forgot to include this in my commit:
```suggestion
exec curl -o /dev/null -kLs "https://0:6443/healthz"
```
We can't use CA cert here as its valid for `api.` domain only